AdyenCard/Form/FormCardExpiryDateItemView.swift

```diff
@@ -0,0 +1,30 @@
+///
+/// Copyright (c) 2021 Adyen N.V.
+///
+/// This file is open source and available under the MIT license. See the LICENSE file for more info.
+///
+
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
+/// A view representing a form card expiry date item.
+internal final class FormCardExpiryDateItemView: FormTextItemView<FormCardExpiryDateItem> {
+    
+    internal required init(item: FormCardExpiryDateItem) {
+        super.init(item: item)
+
+        setupAccessibility()
+        observe(item.$placeholder) { [weak self] _ in
+            self?.setupAccessibility()
+        }
+        observe(item.$title) { [weak self] _ in
+            self?.setupAccessibility()
+        }
+    }
+
+    private func setupAccessibility() {
+        let title = item.title ?? ""
+        let placeholder = item.placeholder ?? "MM/YY"
+        textField.accessibilityLabel = title.isEmpty ? placeholder : "\(title), \(placeholder)"
```

Comment:
This logic can be calculated on the item (viewModel) and passed here as `item.accesibilityLabel`, wdyt?

Reply:
Oh now I see this is a pandora box really. 
We have few `accessibilityLabel` parameters in:
- `SelectableFormItem`
- `ListItem`
And we have `identifier` in lots of Items but nowhere in a base class.
I see your point, but at the same time adding another `accessibilityLabel` to FormTextInputItem doesn't feel alright while this should belong to a base Form class, what do you think?
I'm open to changes but I'd rather make a change that lasts and doesn't require more refactors in the future.